Framework/Core/test/test_BoostSerializedProcessing.cxx

```diff
@@ -17,6 +17,7 @@
 #include "Framework/CallbackService.h"
 #include "FairMQLogger.h"
 #include "Framework/SerializationMethods.h"
+#include "DataFormatsMID/Cluster2D.h"
```

Comment:
This dependency should be avoided. Better define a local structure for the test

Reply:
Got it.